### ISSUE TITLE
 Single click Heroku deploy option

### ISSUE THREAD

**Issue by pranay01** [SigNoz/signoz#43]
Providing a single click Heroku deploy option may be a good idea to make it easy to test SigNoz for developers.

Tasks
- Check if current SigNoz docker can run in Heroku free tier and the compute requirements are served in the free tier of Heroku. Goal is not to test with high load but to get the application up and running with sample Hot ROD application

**Comment by nipun1999** [comment]
@pranay01 can you assign this ticket to me. It looks like an interesting first issue to start with.

**Comment by pranay01** [comment]
Sure @nipun1999

**Comment by nipun1999** [comment]
I don't think this will be feasible on Heroku since the free tier only allows 512 MB of dyno and we need more than that. Also, there is not any support of docker-compose on Heroku. You can run individual docker files but running multiple images in a free tier is not really possible. Maybe we can have a test small AWS instance running which devs can use to test our product.

**Comment by pranay01** [comment]
@nipun1999 Does paid version of Heroku allow running multiple docker images via Dcoker compose? https://devcenter.heroku.com/articles/local-development-with-docker-compose

Also, Digital ocean 1 click would also be a good deployment option to enable. Have a look at https://github.com/SigNoz/signoz/issues/284

**Comment by Nageshbansal** [comment]
After evaluating Heroku as a potential runtime platform for SigNoz, we found:

* Offering SigNoz as a Heroku add-on would require significant codebase changes and would lock us into Heroku’s policies and compliance rules.
* Providing a one-click “try-it-now” deployment is no longer practical since Heroku has discontinued its free tier.

Given these constraints, Heroku support no longer aligns with our goals or current scope, so we’re closing this issue.
